README.rst

```diff
@@ -0,0 +1,78 @@
+
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-DS18X20/badge/?version=latest
+
+    :target: https://circuitpython.readthedocs.io/projects/ds18x20/en/latest/
+
+    :alt: Documentation Status
+
+.. image :: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
+    :alt: Discord
+
+CircuitPython driver for Dallas 1-Wire temperature sensor.
+
+Dependencies
+=============
+This driver depends on:
+
+* `Adafruit CircuitPython <https://github.com/adafruit/circuitpython>`_
+* `Adafruit OneWire <https://github.com/adafruit/Adafruit_CircuitPython_OneWire>`_
+
+Please ensure all dependencies are available on the CircuitPython filesystem.
+This is easily achieved by downloading
+`the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
+
+Usage Example
+=============
+
+.. code-block:: python
+
+    >>> import board
+    >>> from adafruit_onewire.bus import OneWireBus
+    >>> from adafruit_ds18x20 import DS18X20
+    >>> ow_bus = OneWireBus(board.D2)
+    >>> ds18 = DS18X20(ow_bus, ow_bus.scan()[0])
+    >>> ds18.temperature
+    27.9375
```

Comment:
How about removing the prompt so you can copy it all into a file to run? It'd also be good in a separate file under an examples folder as well.

Reply:
Good idea. I like the idea of examples folder. Allows for demonstrating other and more complex uses of the driver.